Intersect (Core)/Configuration/ClientConfiguration.cs

```diff
@@ -162,6 +162,11 @@ public void Validate()
         /// </summary>
         public bool EnableContextMenus { get; set; } = true;
 
+        /// <summary>
+        /// Configures whether the highlighted even rows of list elements should be marked differently or not.
+        /// </summary>
+        public bool MarkListEvenRows { get; set; } = false;
```

Comment:
This accessibility feature should be default _on_.

Reply:
this feature is not included in that PR's preview.
[i've just explained here why they are _not that different - ( but they are _different_ ! )_ ](https://github.com/AscensionGameDev/Intersect-Engine/pull/1695#issuecomment-1369255731)

Reply:
that being said, i will change it to enabled by default within this PR 👍🏽